api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/cognito/AmazonCognitoAuthenticationProperties.java

```diff
@@ -56,4 +60,30 @@
      */
     @RequiredProperty
     private String userPoolId;
+
+    /**
+     * Describes the map of attributes that are to be fetched from the Cognito user pool (map keys)
```

Comment:
This seems inaccurate. You're not describing a Map here. The description should change to indicate that the type is a List, and that there are no map keys or values because there is no Map.

Reply:
I will update the description.